genesyscloud/resource_genesyscloud_routing_queue.go

```diff
@@ -1184,21 +1091,10 @@ func buildSdkConditionalGroupRouting(d *schema.ResourceData) (*platformclientv2.
 			}
 			var sdkCGRRule platformclientv2.Conditionalgrouproutingrule
 
-			if waitSeconds, ok := ruleSettings["wait_seconds"].(int); ok {
-				sdkCGRRule.WaitSeconds = &waitSeconds
-			}
-
-			if operator, ok := ruleSettings["operator"].(string); ok {
-				sdkCGRRule.Operator = &operator
-			}
-
-			if conditionValue, ok := ruleSettings["condition_value"].(float64); ok {
-				sdkCGRRule.ConditionValue = &conditionValue
-			}
-
-			if metric, ok := ruleSettings["metric"].(string); ok {
-				sdkCGRRule.Metric = &metric
-			}
+			sdkCGRRule.WaitSeconds = platformclientv2.Int(ruleSettings["wait_seconds"].(int))
```

Comment:
There was logic before to check if `wait_seconds` existed in the map and to set it on the object if it did. It doesn't look like the Int function in the sdk provides that. It'd be grand if the field was required because that would guarantee the value is set, but in this case a panic could occur.

I tested it out with the code below. If `c` isn't added to the map, you get a panic. If it is, everything runs fine.

```golang
package main

import (
	"fmt"

	"github.com/mypurecloud/platform-client-sdk-go/v115/platformclientv2"
)

type Foo struct {
	Bar *int
}

func main() {
	var x = make(map[string]interface{})

	x["c"] = 3

	var f Foo

	f.Bar = platformclientv2.Int(x["c"].(int))
        if f.Bar != nil {
		fmt.Println(*f.Bar)
        }
}

```